.gitmodules

```diff
@@ -41,3 +37,6 @@
 [submodule "cmake/external/spdlog"]
 	path = cmake/external/spdlog
 	url = https://github.com/gabime/spdlog.git
+[submodule "cmake/external/onnx-tensorrt"]
```

Comment:
why was branch = v5.0 removed? even if we move to 5.1, we should target a branch.
the CI is still targeting TRT 5.0 anyway.

Reply:
Nivida only put latest bug fixes in the master branch and branch 5.0 is not updated. I need to ask them to create a new branch or merge the bug fixes to branch 5.0.